src/styles/blocks/page.scss

```diff
@@ -0,0 +1,12 @@
+html {
+  color: $secondary-text-color;
```

Comment:
The variable `$secondary-text-color` is used here, but it's not defined in this file. Ensure that this variable is defined in a parent file or imported correctly to avoid compilation errors.